sql/driver/wire.go

```diff
@@ -62,6 +66,23 @@ func makeDatum(val driver.Value) (Datum, error) {
 	return datum, nil
 }
 
+// Date wraps time.Time and provides a custom String() method.
+type Date struct {
+	time.Time
+}
+
+// MakeDate constructs a Date from a time.Time.
+func MakeDate(t time.Time) Date {
+	year, month, day := t.Date()
+	return Date{Time: time.Date(year, month, day, 0, 0, 0, 0, t.Location())}
```

Comment:
Is this the same as `return t.Truncate(24 * time.Hour)`? If not, how is it different?